test/lint.test.js

```diff
@@ -0,0 +1,11 @@
+import lint from 'mocha-eslint';
```

Comment:
IMO, better to add a `lint` command to package.json, then have another command (I usually use `check` that does `yarn run lint && yarn test`, which is the command run by CI. Thoughts?